### PR TITLE
Issue #45352 Update supported configurations file.

### PR DESCRIPTION
**Pull request by JStickler**
Addresses GitHub issue #45352

Version(s): 4.6 - 4.11

Link to docs preview: https://deploy-preview-45442--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/preparing-ossm-installation.html#ossm-supported-configurations_preparing-ossm-installation
AND
https://deploy-preview-45442--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v1x/preparing-ossm-installation.html#ossm-supported-configurations-v1x_preparing-ossm-installation-v1x

QE review - mattmahoneyrh
Peer review - rolfedh

### CODE REVIEW

**Comment by mattmahoneyrh** [comment]
LGTM

**Review by rolfedh — COMMENTED** [review]
- Please add `- Aligned team: Service Mesh` to the PR description.

File: modules/ossm-supported-configurations.adoc [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
+* {product-dedicated} version 4.
 * Azure Red Hat OpenShift version 4.
 
 [NOTE]
 ====
-Red Hat OpenShift Online is not supported for {SMProductName}.
+OpenShift Online is not supported for {SMProductName}.
 ====
 
 * This release of {SMProductName} is only available on {product-title} x86_64, IBM Z, and IBM Power Systems.
```
Comment (rolfedh) [inline modules/ossm-supported-configurations.adoc:21]: When I read the doc preview of this topic, the logic behind this list seems to "fall apart" here on line 21.  Should we move this line and its two sub-bullets out of this list?
The result would look like this: 
```
This release of {SMProductName} is only available on {product-title} x86_64, IBM Z, and IBM Power Systems.
* IBM Z is only supported on {product-title} 4.6 and later.
* IBM Power Systems is only supported on {product-title} 4.6 and later.```

Comment (rolfedh) [reply modules/ossm-supported-configurations.adoc:21]: Same issue in `modules/ossm-supported-configurations-v1x.adoc`

**Review by rolfedh — APPROVED** [review]
LGTM. 
I raised some separate concerns about the content with Julie, who will create a Jira task to address them.

**Pull request merged**